RayleighTaylorInstability/dash_functions_RTI.jl

```diff
@@ -1,5 +1,90 @@
 using DelimitedFiles
 
+
+"""
+Returns an accordion menu containing the rheological parameters.
+"""
+function make_rheological_parameters()
+    return dbc_accordionitem(title="Rheological Parameters", [
+        make_accordion_item("η_up(log₁₀(Pa⋅s)):", "viscosity_upper", "Logarithm of the viscosity of the upper layers.", 21.0, 16.0, 23.0),
+        dbc_row(html_p()),
+        make_accordion_item("η_lo(log₁₀(Pa⋅s)):", "viscosity_lower", "Logarithm of the viscosity of the lower layer", 20.0, 16.0, 23.0),
+        dbc_row(html_p()),
+        make_accordion_item("ρ_up:", "density_upper", "Density of the upper layers.", 2800.0, 2.0, 5000.0),
+        dbc_row(html_p()),
+        make_accordion_item("ρ_lo:", "density_lower", "Density of the lower layer.", 2200.0, 2.0, 5000.0),
+    ])
+end
+
+"""
+Returns an accordion menu containing the simulation parameters.
+"""
+function make_simulation_parameters()
+    return dbc_accordionitem(title="Simulation Parameters", [
+        make_accordion_item("Width (km):", "domain_width", "Width of the domain, given in kilometers.", 10.0, 1.0e-10),
+        dbc_row(html_p()),
+        make_accordion_item("Depth of the interface (km):", "depth", "Depth of the interface, given in kilometers.", -2.5, -50.0),
+        dbc_row(html_p()),
+        make_accordion_item("nx:", "nel_x", "Number of elements in the x-direction. Must be an integer greater than 2.", 64, 2),
+        dbc_row(html_p()),
+        make_accordion_item("nz:", "nel_z", "Number of elements in the z-direction. Must be an integer greater than 2.", 32, 2),
+        dbc_row(html_p()),
+        make_accordion_item("nt:", "n_timesteps", "Maximum number of timesteps. Must be an integer greater than 1.", 50, 1),
+        dbc_row(html_p()),
+        dbc_row([
+            dbc_checklist(options=["FreeSurf"],
+                    id="switch-FreeSurf",
+                    switch=true,
+            )
+        ]),
+        dbc_row(html_p()),
+        dbc_row([
+            dbc_checklist(options=["Layers"],
+                    id="switch-Layers",
+                    switch=true,
+            )
+        ])
+    ])
+end
+
+"""
+Creates a setup with noisy temperature and one phase
+"""
+function CreateSetup(ParamFile, layered_overburden=false, Hi=-5.0, ampl_noise=0.1, ; args)
+    Grid        =   ReadLaMEM_InputFile(ParamFile, args=args)
+    Phases      =   zeros(Int64, size(Grid.X));      
+    Temp        =   zeros(Float64,size(Grid.X));  
+    
+
+    if layered_overburden
+        H_layer = 0.25;
+        for z_low = minimum(Grid.Z):2*H_layer:maximum(Grid.Z)
+            # print(z_low)
+            # z_low = -z_low
+            iz = findall( (Grid.Z[1,1,:] .> z_low) .&  (Grid.Z[1,1,:] .<= (z_low+H_layer) )) 
```

Comment:
there's no need to use `findall`, you can simply remove it and then `iz` will be a bitmask